### ISSUE TITLE
INDILIB cmake error with ARM_64

### ISSUE THREAD

**Issue by Andre87osx** [Andre87osx/AstroPi-system#54]
INDI3dparty compiler error with ARM_64 enable

**Comment by Andre87osx** [comment]
use in cmake 
`-DWITH_LIMESDR=Off`
`-DAHP_CORRELATOR=Off`

**Comment by Andre87osx** [comment]
cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWITH_FXLOAD=1 $USERHOME/position/indi-3rdparty

**Comment by Andre87osx** [comment]
-DCMAKE_CXX_FLAGS=-m32 -DCMAKE_C_FLAGS=-m32

(-DUSE32bit=true)

**Comment by Andre87osx** [comment]
```
sudo apt purge binutils
sudo apt remove make
sudo apt autoremove
sudo apt install build-essential
```

**Comment by Andre87osx** [comment]
https://gitmemory.com/issue/indilib/indi-3rdparty/439/907781697

**Comment by Andre87osx** [comment]
fix in 25f093f
